sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/CryptographyConstants.java

```diff
@@ -15,7 +18,13 @@
 
     static final String AES = "AES";
 
-    static final String AGENT_METADATA_VALUE = "JavaTrack2" + BlobCryptographyConfiguration.VERSION;
+    static final String AZURE_STORAGE_PROPERTIES = "azure-storage-blob.properties";
+
+    static final String BLOB_CRYPTOGRAPHY_NAME = getName();
+
+    static final String BLOB_CRYPTOGRAPHY_VERSION = getVersion();
+
+    static final String AGENT_METADATA_VALUE = "JavaTrack2" + BLOB_CRYPTOGRAPHY_VERSION;
```

Comment:
After the above change is made this will be:

```java
    static final String AGENT_METADATA_VALUE = "JavaTrack2" + AZURE_STORAGE_PROPERTIES.getVersion();

```

Reply:
Done.